### PR TITLE
[ClickHouse] Add Travis CI check for ClickHouse

### PR DESCRIPTION
**Pull request by qoega**
Also
- Added TLPAggregateOracle, TLPDistinctOracle
- DISTINCT is supported in SELECT query
- TLPHaving is disabled for now

### CODE REVIEW

**Comment by mrigger** [comment]
Thanks a lot for the PR! I'll review it tomorrow.

File: test/sqlancer/dbms/TestClickHouse.java [hunk 1]
```diff
@@ -6,19 +6,18 @@
...
-    public void testClickHouseTLPWhereHaving() {
+    public void testClickHouseTLPWhereGroupBy() {
         String clickHouseAvailable = System.getenv("CLICKHOUSE_AVAILABLE");
         boolean clickHouseIsAvailable = clickHouseAvailable != null && clickHouseAvailable.equalsIgnoreCase("true");
         assumeTrue(clickHouseIsAvailable);
         Assertions.assertEquals(0,
-                Main.executeMain(new String[] { "--timeout-seconds", TestConfig.SECONDS, "--num-queries",
-                        TestConfig.NUM_QUERIES, "--num-threads", "50", "clickhouse", "--oracle", "TLPWhere", "--oracle",
-                        "TLPHaving" }));
+                Main.executeMain(new String[] { "--timeout-seconds", TestConfig.SECONDS, "--num-queries", "1000",
```
Comment (mrigger) [inline test/sqlancer/dbms/TestClickHouse.java:18]: Could you please use https://github.com/sqlancer/sqlancer/blob/e62224edcb43fe421dc87601f75090739e34e595/test/sqlancer/dbms/TestConfig.java#L4 here?

Comment (qoega) [reply test/sqlancer/dbms/TestClickHouse.java:18]: Ok. It left after it was set to 0 with TLPHaving oracle enabled.

**Comment by mrigger** [comment]
It might be a good idea to create smaller-sized PRs, so that we can merge them more quickly. In case my recent refactorings have created a merge conflict, I'm sorry about it!

In a future PR (or in this one) you might consider splitting the ClickHouse Travis CI jobs (or reducing their run time). It seems that with this change, the ClickHouse job takes ~25 minutes, while the other jobs take up to ~13 minutes.

**Comment by qoega** [comment]
I totally understand. Actually several changes were like avalanche when I tried to fix problems found in CI runs.
Currently I have yet another bug report that triggers often in Travis but rarely on my server: https://github.com/ClickHouse/ClickHouse/issues/12402

I've already disabled two checks and reduced test runs to 1 minute per oracle. We plan to run longer tests in out CI infrastucture.

**Pull request merged**